src/V3EmitCModel.cpp

```diff
@@ -387,6 +387,7 @@ class EmitCModel final : public EmitCFunc {
         if (v3Global.opt.trace()) puts("vlSymsp->__Vm_activity = true;\n");
 
         if (v3Global.hasEvents()) puts("vlSymsp->clearTriggeredEvents();\n");
+        puts("vlSymsp->__Vm_deleter.deleteAll();\n");
```

Comment:
Would be good to add a `if (v3Global.hasClasses())` so we don't get this when not needed, like the hasEvents stuff above. Just set `hasClasses` true if parsed a class.